build.sh

```diff
@@ -5,11 +5,156 @@
 # This script is basically automation script of dev.md
 # It is supposed to run in same folder where the file lives (root).
 
-# 1. enter into build folder
-mkdir -p build && cd build
+# Unfortunately we still need to manually fix the GRPC issue listed in dev.md
+
+if [ "$(expr substr $(uname -s) 1 5)" != "Linux" ]; then
```

Comment:
nit: the command itself won't work on macos, I think just $(uname -s) itself should be fine.